tg.js

```diff
@@ -308,12 +308,12 @@ settings.gameprivacy.header Pagkapribado ng Laro
 settings.gameprivacy.streamer Mode ng Streamer
```

Comment:
```suggestion
settings.gameprivacy.streamer Streamer Mode
```
for uniformity with anonymous mode